test/kitchen/tasks/kitchen.py

```diff
@@ -37,7 +38,23 @@ def genconfig(
     if not platlist and not provider:
         provider = "azure"
 
-    platforms = load_platforms(ctx, platformfile=platformfile)
+    if platformfile is not None:
+        platforms = load_platforms(ctx, platformfile=platformfile)
+    else:
+        try:
+            print(
+                "Fetching the latest kitchen platforms.json from Github. Use --platformfile=platforms.json to override with a local file."
+            )
+            r = requests.get(
+                'https://raw.githubusercontent.com/DataDog/datadog-agent/master/test/kitchen/platforms.json',
+                allow_redirects=True,
+            )
+            open('/tmp/platforms.json', 'wb').write(r.content)
+            platforms = load_platforms(ctx, platformfile='/tmp/platforms.json')
```

Comment:
Why not simply this?
```suggestion
            platforms = json.load(r.content)
```

Reply:
I removed the `load_platforms` altogether since I didn't want to have some code paths using it and some not.